src/screens/CreatePost/index.js

```diff
@@ -706,7 +706,7 @@ const CreatePost = () => {
 
   React.useEffect(() => {
     const followType = params?.followType;
-    if (followType === 'incognito' || followType === undefined) {
+    if (followType === 'incognito') {
       setTypeUser(true);
     } else {
       setTypeUser(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the condition for setting `typeUser` seems to be a logical improvement. Previously, if `followType` was undefined, `typeUser` would be set to true, which might not have been the intended behavior. Now, `typeUser` is only set to true when `followType` equals 'incognito', which seems more appropriate.
```diff
-    if (followType === 'incognito' || followType === undefined) {
+    if (followType === 'incognito') {
       setTypeUser(true);
     } else {
       setTypeUser(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->